knnvector/README.md

```diff
@@ -0,0 +1,169 @@
+# KNN Vector Workload
+
+This workload is to benchmark performance of indexing and search using knn_vector as field type.
+
+## Datasets
```

Comment:
I don't see any sample commands to run the workload. Lets make sure that we add some sample commands and format on how to run the workload.

Reply:
This is just workload repository. OSB invokes this repository, and i believe dependencies should be added to OSB. i asked this question here https://github.com/opensearch-project/opensearch-benchmark-workloads/pull/144#issuecomment-1823675797 , waiting for response.

Reply:
Moved all dependencies to opensearch-benchmark repository and added section on how to run osb